city_scrapers/mixins/wayne_commission.py

```diff
@@ -11,6 +11,11 @@
 class Wayne_commission:
     timezone = 'America/Detroit'
     allowed_domains = ['www.waynecounty.com']
+    location = {
+        'name': '7th floor meeting room, Guardian Building',
+        'address': '500 Griswold St, Detroit, MI 48226',
+        'neighborhood': '',
+    }
 
     # Calendar shows only meetings in current year.
     yearStr = datetime.now().year
```

Comment:
This is a small thing, but it would be good to change this to `year_str` for style consistency and move it into `_parse_start` as a normal variable since that's the only place it's referenced